geospaas/catalog/models.py

```diff
@@ -33,6 +33,11 @@ class GeographicLocation(geomodels.Model):
     def __str__(self):
         return str(self.geometry.geom_type)
 
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(name='unique_geographic_location', fields=['geometry'])
```

Comment:
So it works this way both in spatialite and PostGIS?

Reply:
Yes, but I had to do some tinkering with the migration to make it work: I manually add a unique index when the database is sqlite (according to sqlite [documentation](https://www.sqlite.org/lang_createtable.html), it is equivalent to adding a unique constraint).